tests/acceptance/cli_test.py

```diff
@@ -467,6 +467,35 @@ def test_ps_alternate_composefile(self):
         self.assertNotIn('multiplecomposefiles_another_1', result.stdout)
         self.assertIn('multiplecomposefiles_yetanother_1', result.stdout)
 
+    def test_ps_services_filter_option(self):
+        self.base_dir = 'tests/fixtures/ps-services-filter'
+        image = self.dispatch(['ps', '--services', '--filter', 'key=image'])
+        build = self.dispatch(['ps', '--services', '--filter', 'key=build'])
+        all_services = self.dispatch(['ps', '--services'])
+
+        self.assertIn('with_build', all_services.stdout)
```

Comment:
Please use the more readable format `assert 'with_build' in all_services.stdout` for newly added tests!

Reply:
Done!